project-service/api.md

```diff
@@ -0,0 +1,594 @@
+# Project Service External API
+
+## Users
+
+### Fetch Users
+
+Returns json data of all users as an array
+
+- URL
+  + `/users`
```

Comment:
I'll leave this out for now, since we probably wouldn't get to autocomplete invitation. Right now the api allows adding users via userId, but we can change that to use email for now.